upgrade/1.2/Stage_0_Prerequisites.md

```diff
@@ -26,6 +26,12 @@ backup of Workload Manager configuration data and files is created. Once complet
     CSM_RELEASE=1.2.0
    ```
 
+1. If there are space concerns on the node, then add an `rbd` device on the node for the CSM tarball.
+
+    This same `rbd` device can be remapped to `ncn-m002` later in the upgrade procedure, when the CSM tarball is needed on that node.
+
```

Comment:
```suggestion

    **Note:** The `prepare-assets.sh` script will delete the CSM tarball. Copy it to a different location before running that script.
    Otherwise it will need to be downloaded again later in the upgrade procedure.

```